setup.cfg

```diff
@@ -1,2 +1,2 @@
 [pycodestyle]
-select = E3
+max-line-length = 119
```

Comment:
I chose this number mostly because it's used in Django repo I'm used to and IMHO it's a reasonable number.